configurations/pipeline_config.json

```diff
@@ -1,4 +1,50 @@
 {
   "RapidProBaseURL": "https://textit.in",
-  "RapidProTokenFileURL": "gs://avf-credentials/icraf-text-it-token.txt"
+  "RapidProTokenFileURL": "gs://avf-credentials/icraf-text-it-token.txt",
+  "RapidProKeyRemappings":[
+    {"RapidProKey": "avf_phone_id", "PipelineKey": "uid"},
+
+    {"RapidProKey": "Rqa_S01_E01 (Run ID) - icraf_s01_e01_activation", "PipelineKey": "rqa_s01e01_run_id"},
+    {"RapidProKey": "Rqa_S01_E02 (Run ID) - icraf_s01_e02_activation", "PipelineKey": "rqa_s01e02_run_id"},
+    {"RapidProKey": "Rqa_S01_E03 (Run ID) - icraf_s01_e03_activation", "PipelineKey": "rqa_s01e03_run_id"},
+    {"RapidProKey": "Rqa_S01_E04 (Run ID) - icraf_s01_e04_activation", "PipelineKey": "rqa_s01e04_run_id"},
+    {"RapidProKey": "Rqa_S01_E05 (Run ID) - icraf_s01_e05_activation", "PipelineKey": "rqa_s01e05_run_id"},
+    {"RapidProKey": "Rqa_S01_E06 (Run ID) - icraf_s01_e06_activation", "PipelineKey": "rqa_s01e06_run_id"},
+
+    {"RapidProKey": "Rqa_S01_E01 (Time) - icraf_s01_e01_activation", "PipelineKey": "sent_on"},
+    {"RapidProKey": "Rqa_S01_E02 (Time) - icraf_s01_e02_activation", "PipelineKey": "sent_on"},
+    {"RapidProKey": "Rqa_S01_E03 (Time) - icraf_s01_e03_activation", "PipelineKey": "sent_on"},
+    {"RapidProKey": "Rqa_S01_E04 (Time) - icraf_s01_e04_activation", "PipelineKey": "sent_on"},
+    {"RapidProKey": "Rqa_S01_E05 (Time) - icraf_s01_e05_activation", "PipelineKey": "sent_on"},
+    {"RapidProKey": "Rqa_S01_E06 (Time) - icraf_s01_e06_activation", "PipelineKey": "sent_on"},
+
+    {"RapidProKey": "Gender (Value) - icraf_s01_demogs", "PipelineKey": "gender_raw"},
+    {"RapidProKey": "Age (Value) - icraf_s01_demogs", "PipelineKey": "age_raw"},
+    {"RapidProKey": "Constituency (Value) - icraf_s01_demogs", "PipelineKey": "constituency_raw"},
+    {"RapidProKey": "Livelihood (Value) - icraf_s01_demogs", "PipelineKey": "livelihood_raw"},
+
+    {"RapidProKey": "Gender (Time) - icraf_s01_demogs", "PipelineKey": "gender_time"},
+    {"RapidProKey": "Age (Time) - icraf_s01_demogs", "PipelineKey": "age_time"},
+    {"RapidProKey": "Constituency (Time) - icraf_s01_demogs", "PipelineKey": "constituency_time"},
+    {"RapidProKey": "Livelihood (Time) - icraf_s01_demogs", "PipelineKey": "livelihood_time"},
+
+    {"RapidProKey": "Current_Practices (Run ID) - icraf_s01_follow_up_survey", "PipelineKey": "current_practices_run_id"},
```

Comment:
Why do you need the run id for follow-up surveys?

Reply:
Removed thanks